packages/manager/modules/pci/src/projects/project/inactive/inactive.html

```diff
@@ -0,0 +1,16 @@
+<oui-modal
+    data-on-dismiss="$ctrl.goBack()"
+    data-heading="{{ $ctrl.title }}"
+    data-type="warning"
+    primary-label="{{ ::($ctrl.projectData.hasPendingDebt
+            ? ('pci_projects_project_pay_bill' | translate)
+            : ($ctrl.projectData.isSuspended ? ('pci_projects_project_inactive_close' | translate) : '')
+        ) }}"
+    primary-action="$ctrl.projectData.hasPendingDebt ? $ctrl.goToBilling() : $ctrl.goBack()"
+    secondary-label="{{ ::(($ctrl.projectData.hasPendingDebt && !$ctrl.projectData.isSuspended) ? ('pci_projects_project_view' | translate) : '') }}"
```

Comment:
The secondary action label can have empty value ? Is it normal ?

Reply:
When it has an empty value, the secondary button would not come up. And that is what we need :)